camkes/templates/component.environment.c

```diff
@@ -1,13 +1,7 @@
 /*#
- *# Copyright 2018, Data61
- *# Commonwealth Scientific and Industrial Research Organisation (CSIRO)
- *# ABN 41 687 119 230.
- *#
- *# This software may be distributed and modified according to the terms of
- *# the BSD 2-Clause license. Note that NO WARRANTY is provided.
- *# See "LICENSE_BSD2.txt" for details.
- *#
- *# @TAG(DATA61_BSD)
+ * Copyright 2018, Data61, CSIRO (ABN 41 687 119 230)
```

Comment:
Not sure if this has an observable effect, but in the old change, the inner bullet char sequence was `*#`

Reply:
Should not be observable (the comment gets stripped). I had to to a bit of manual search and replace, so the `*` vs `*#` inner sequences are not 100% the same. I think that's Ok.